src/mruby.rs

```diff
@@ -533,7 +533,7 @@ pub trait MrubyImpl {
     ///
     /// ```
     /// # #[macro_use] extern crate mrusty;
-    /// use mrusty::*;
+    /// use mrusty::{Mruby, MrubyFile, MrubyImpl, MrubyType};
```

Comment:
I think the idea has merit (per issue #85), but I honestly don't see a major difference between `use mrusty::*` and `use mrusty::prelude::*`. Seems like `prelude` might be a growing standard, but I also think `mrusty` has *just* few enough total exports that a recommended prelude would essentially be everything in the crate!

For these tests, I explicitly opted out of glob imports to try and ensure all the macros worked properly.